manifests/init.pp

```diff
@@ -13,8 +13,20 @@
 #
 # $group::                         Override the name of the puppet group.
 #
+# $dir_prefix::                    Override the where to find the data directory on Windows.
```

Comment:
I don't think this is used in the class, only inside params.  Does it need to be a parameter, or should we join the paths inside the class instead of params?